modules/deleteBookRecord.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable import/prefer-default-export */
+export const deleteBookRecord = (bookList, e) => {
+  const deleteButton = e.target;
+  if (
+    deleteButton !== undefined
+        && deleteButton.classList.contains('delete')
+  ) {
+    e.preventDefault();
+    const tr = deleteButton.closest('tr');
+    const isbn = tr.getAttribute('data-isbn');
+    tr.remove();
+
+    bookList.removeNewBook(isbn);
+  }
+};
```

Comment:
- [ ] It is not possible to delete books, after page refresh all books are listed back. Please try to delete books and check your console:
<img width="476" alt="Screenshot 2022-03-22 at 20 41 12" src="https://user-images.githubusercontent.com/77038610/159530733-cef2ec9f-0bda-4f47-a4cf-e7835f5a4cf5.png">

- [ ] Please make sure to address this issue 🙏🏻